db/seeds.rb

```diff
@@ -5,3 +5,11 @@
 #
 #   movies = Movie.create([{ name: 'Star Wars' }, { name: 'Lord of the Rings' }])
 #   Character.create(name: 'Luke', movie: movies.first)
+
+20.times do 
+    Post.create(
+        post_title: Faker::Name.post_title,
```

Comment:
what is `Name.post_title`? please check documentation of faker.

Reply:
Sheet Happens. I was understand wrong about Faker Attribute. Thanks @sadmanahmed for helping me. :) Checl latest commit. :)